src/gatsby/src/components/search/filter.js

```diff
@@ -58,10 +58,21 @@ const Filter = ({ filterItems, selectedFilterItems, handleFilterCheck, filterTyp
               checked={checked}
               onChange={event => {
                 handleFilterCheck(item, event)
+                if (typeof window.snowplow === 'function') {
+                  window.snowplow(
+                    "trackStructEvent",
+                    "Checkbox",
+                    event.target.checked ? "Check" : "Uncheck",
```

Comment:
The action should capture what UI element was interacted with and what the user did. 
e.g. `filter-select`, `filter-unselect`